RHEL/7/input/oval/dconf_gnome_disable_ctrlaltdel_reboot.xml

```diff
@@ -0,0 +1,46 @@
+<def-group>
+  <definition class="compliance" id="dconf_gnome_disable_ctrlaltdel_reboot" version="1">
+    <metadata>
+      <title>Disable Ctrl-Alt-Del Reboot Key Sequence in GNOME3</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+      </affected>
+      <description>Disable the GNOME3 ctrl-alt-del reboot key sequence in GNOME3.</description>
+      <reference source="galford" ref_id="20160415" ref_url="test_attestation" />
+    </metadata>
+    <criteria operator="OR">
+      <extend_definition comment="dconf installed" definition_ref="package_dconf_installed" negate="true" />
+      <criteria operator="AND">
+        <extend_definition comment="dconf user profile exists" definition_ref="enable_dconf_user_profile" />
+        <criterion comment="Disable Ctrl-Alt-Del" test_ref="test_disable_gnome_ctrlaltdel" />
+        <criterion comment="Prevent user from changing" test_ref="test_prevent_user_enable_ctrlaltdel" />
+      </criteria>
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist"
+  comment="Disable Ctrl-Alt-Del"
+  id="test_disable_gnome_ctrlaltdel" version="1">
+    <ind:object object_ref="obj_disable_gnome_ctrlaltdel" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="obj_disable_gnome_ctrlaltdel"
+  version="1">
+    <ind:path>/etc/dconf/db/local.d/</ind:path>
```

Comment:
Actually taking back, the proper ```/etc/dconf/db/local.d/00-security-settings``` content is:
```
# cat /etc/dconf/db/local.d/00-security-settings 
[org/gnome/settings-daemon/plugins/media-keys]
logout=''
```
not
```
# cat /etc/dconf/db/local.d/00-security-settings 
[org/gnome/settings-daemon/plugins/media-keys/logout]
logout=''
```
I have had there before.

After correction, this seems to be working fine. Sorry for turmoil (=> ignore my previous comment).